cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -767,7 +765,7 @@ func TestGetVolumeStatCapabilitiesWithVmMode(t *testing.T) {
 }
 
 func TestPublishDeviceWithReadWriteManyModeIsNotSupportedWithNBS(t *testing.T) {
-	tempDir := os.TempDir()
+	tempDir := t.TempDir()
 
 	nbsClient := mocks.NewNbsClientMock()
 	mounter := csimounter.NewMock()
```

Comment:
WithoutVmMode shouldn't create NBD device. This is not directly related to this PR, but I just noticed it during review.

Reply:
Removed